// modul8/test/clash/app/entry.js
exports.app_safe0slash = require('safe0/');
exports.app_safe0 = require('safe0');
exports.app_unsafe0slash = require('unsafe0/');
exports.app_unsafe0 = require('unsafe0');
exports.app_clash0slash = require('clash0/');
exports.app_clash0 = require('clash0');
exports.app_revclash0slash = require('revclash0/');
exports.app_revclash0 = require('revclash0');
exports.app_safe1slash = require('safe1/');
exports.app_safe1 = require('safe1');
exports.app_unsafe1slash = require('unsafe1/');
exports.app_unsafe1 = require('unsafe1');
exports.app_clash1slash = require('clash1/');
exports.app_clash1 = require('clash1');
exports.app_revclash1slash = require('revclash1/');
exports.app_revclash1 = require('revclash1');
exports.shared_safe0slash = require('shared::safe0/');
exports.shared_safe0 = require('shared::safe0');
exports.shared_unsafe0slash = require('shared::unsafe0/');
exports.shared_unsafe0 = require('shared::unsafe0');
exports.shared_clash0slash = require('shared::clash0/');
exports.shared_clash0 = require('shared::clash0');
exports.shared_revclash0slash = require('shared::revclash0/');
exports.shared_revclash0 = require('shared::revclash0');
exports.shared_safe1slash = require('shared::safe1/');
exports.shared_safe1 = require('shared::safe1');
exports.shared_unsafe1slash = require('shared::unsafe1/');
exports.shared_unsafe1 = require('shared::unsafe1');
exports.shared_clash1slash = require('shared::clash1/');
exports.shared_clash1 = require('shared::clash1');
exports.shared_revclash1slash = require('shared::revclash1/');
exports.shared_revclash1 = require('shared::revclash1');
